src/sonal/Assignment1.java

```diff
@@ -0,0 +1,123 @@
+package sonal;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+public class Assignment1 {
+
+
+
+	public static void main(String[] args) throws InterruptedException {
+		
+		WebDriver driver = new ChromeDriver();
+		
+		driver.get("file:///C:/Users/Dell/Downloads/SeleniumAssignment_1.html");
+		
+		String acttitle = driver.getTitle();
+		
+		System.out.println("Title of the webpage is: " +acttitle);  //print the title
+		
+		String exptitle = "TECHNOCREDITS";
+		
+		if (acttitle.equals(exptitle))
+			{
+				System.out.println("Test is Passed: Title is correct");
+			}else
+			
+				System.out.println("Test is Failed: Title is incorrect");
+		
+		driver.findElement(By.id("first name")).sendKeys("Sonal");
+		driver.findElement(By.id("last name")).sendKeys("Inamdar");
+		driver.findElement(By.id("E-mail")).sendKeys("Inamdar@gmail.com");
+		driver.findElement(By.id("Company Name")).sendKeys("Inamdar");
+		
+	
+		
+	WebElement radio1= driver.findElement(By.xpath("//input[@value='male']"));
```

Comment:
Why you have not given name same as radio button name, radio1 & radio2 looks not good at all.